pyvista/utilities/geometric_objects.py

```diff
@@ -465,6 +466,35 @@ def Line(pointa=(-0.5, 0.0, 0.0), pointb=(0.5, 0.0, 0.0), resolution=1):
     return line
 
 
+def MultipleLines(points=[[-0.5, 0.0, 0.0], [0.5, 0.0, 0.0]]):
+    """Create multiple lines.
+
+    Parameters
+    ----------
+    points : np.ndarray or list, optional
+        List of points defining a broken line, default is ``None``
+        If given, pointa, pointb and resolution will be ignored.
+
+    Returns
+    -------
+    pyvista.PolyData
+
+    Examples
+    --------
+    Create a multiple lines between ``(0, 0, 0)``, ``(1, 1, 1)`` and ``(0, 0, 1)``.
+
+    >>> import pyvista
+    >>> mesh = pyvista.MultipleLines(points=[[0, 0, 0], [1, 1, 1], [0, 0, 1]])
+    >>> mesh.plot(color='k', line_width=10)
+    """
+    points = _coerce_pointslike_arg(points)
+    src = _vtk.vtkLineSource()
+    src.SetPoints(pyvista.vtk_points(points))
```

Comment:
Should we check that the number of points is >=2?

Reply:
Thank you for pointing this out. We need points >=2.
```python
>>> multiple_lines = pv.MultipleLines([[1.0, 1.0, 1.0]])
2022-03-13 09:46:08.992 ( 111.636s) [        22663C00]      vtkLineSource.cxx:125   WARN| vtkLineSource (0x3354b50): Cannot define a broken line with given input.
2022-03-13 09:46:08.992 ( 111.636s) [        22663C00]       vtkExecutive.cxx:752    ERR| vtkCompositeDataPipeline (0x348ab70): Algorithm vtkLineSource(0x3354b50) returned failure for request: vtkInformation (0x3381bd0)
  Debug: Off
  Modified Time: 237
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA
  FROM_OUTPUT_PORT: 0
  ALGORITHM_AFTER_FORWARD: 1
  FORWARD_DIRECTION: 0
```

Reply:
```suggestion
    if not (len(points) >= 2):
        raise ValueError('>=2 points need to define multiple lines.')
    src.SetPoints(pyvista.vtk_points(points))
```